plutus-contract/src/Wallet/API.hs

```diff
@@ -104,9 +109,10 @@ payToPaymentPublicKeyHash_ ::
     ( Member WalletEffect effs
     , Member (Error WalletAPIError) effs
     , Member (LogMsg Text) effs
+    , Member (LogMsg RequestHandlerLogMsg) effs
     )
     => SlotRange -> Value -> PaymentPubKeyHash -> Eff effs ()
-payToPaymentPublicKeyHash_ r v = void . payToPaymentPublicKeyHash r v
+payToPaymentPublicKeyHash_ r v = void . payToPaymentPublicKeyHash def r v
```

Comment:
Mmm... not sure here about using `def`. This is not an emulator function, this can actually be used in the `Contract` monad, no?

Reply:
The only user of it in plutus-apps is `Spec.Emulator` module so I thought that it's internal. But yeah, we can introduce an argument here and pass `def` at usage points. Should I?